msu/systems/keybinds/key_static.nut

```diff
@@ -30,7 +30,7 @@ this.MSU.Key <- {
 		if (keyArray.len() > 1)
 		{
 			keyArray.sort();
-			key = keyArray.reduce(@(a, b) a + "+" + b) + "+";
+			key = keyArray.reduce(@(_a, _b) _a + "+" + _b) + "+";
```

Comment:
Lambda function parameters traditionally don't get a _ prefix, I'm afraid you'll have to update your syntax highlighter

Reply:
_No_